src/lib/core/CHIPError.cpp

```diff
@@ -515,7 +527,7 @@ bool FormatCHIPError(char * buf, uint16_t bufSize, CHIP_ERROR err)
     case CHIP_ERROR_TOO_MANY_SHARED_SESSION_END_NODES.AsInteger():
         desc = "Too many shared session end nodes";
         break;
-    case CHIP_ERROR_IM_MALFORMED_ATTRIBUTE_DATA_ELEMENT.AsInteger():
+    case CHIP_ERROR_IM_MALFORMED_ATTRIBUTE_DATA_IB.AsInteger():
         desc = "Malformed Interaction Model Attribute DataElement";
```

Comment:
Again, what is a DataElement (in the description string)?

Reply:
good catch, just review all these description, and correct them, thanks